be/src/vec/exprs/table_function/vexplode.h

```diff
@@ -43,4 +47,13 @@ class VExplodeTableFunction : public TableFunction {
     size_t _pos;
 };
 
+class VExplodeOuterTableFunction : public VExplodeTableFunction {
+public:
```

Comment:
You can use template specific on `TableFunctionCreator<VExplodeTableFunction>` to  reduce a class `VExplodeOuterTableFunction` just like `TableFunctionCreator<VExplodeJsonArrayTableFunction>`.
And so that we can have a unified design style.

Reply:
thanks for your advice, I have updated the codes.